resources/definitions/ultimaker_factor4.def.json

```diff
@@ -0,0 +1,401 @@
+        },
+        "bridge_skin_speed": { "value": "speed_topbottom" },
+        "bridge_skin_speed_2": { "value": "speed_topbottom" },
+        "bridge_skin_speed_3": { "value": "speed_topbottom" },
+        "bridge_wall_coast": { "value": "0" },
+        "bridge_wall_material_flow":
+        {
+            "maximum_value": "100",
+            "value": "wall_material_flow"
+        },
+        "bridge_wall_speed": { "value": "speed_wall" },
+        "brim_width": { "value": "5" },
+        "build_volume_temperature":
+        {
+            "maximum_value": "max(35, min((material_bed_temperature + 20) / 2, 70))",
+            "maximum_value_warning": "max(30, min((material_bed_temperature + 10) / 2, 70))",
+            "minimum_value": "max((material_bed_temperature - 30) / 2, 30)",
+            "minimum_value_warning": "max((material_bed_temperature - 20) / 2, 30)"
+        },
+        "cool_min_layer_time": { "value": 3 },
+        "cool_min_layer_time_fan_speed_max": { "value": "cool_min_layer_time + 12" },
+        "cool_min_speed": { "value": "round(speed_wall_0 * 1 / 2) if cool_lift_head else round(speed_wall_0 / 3)" },
+        "default_material_print_temperature": { "maximum_value": "340" },
+        "expand_skins_expand_distance": { "value": "skin_preshrink * 2" },
+        "extruder_prime_pos_abs": { "default_value": true },
+        "gantry_height": { "value": 35 },
+        "gradual_support_infill_steps": { "value": "3 if support_interface_enable and support_structure != 'tree' else 0" },
+        "group_outer_walls": { "value": "False" },
+        "infill_before_walls": { "value": "False if infill_sparse_density > 50 else True" },
+        "infill_enable_travel_optimization": { "value": "True" },
+        "infill_material_flow":
+        {
+            "maximum_value": "100",
+            "value": "(1 + (skin_material_flow-infill_sparse_density) / 100 if infill_sparse_density > skin_material_flow else 1) * material_flow"
+        },
+        "infill_overlap": { "value": "0" },
+        "infill_pattern": { "value": "'zigzag' if infill_sparse_density > 50 else 'triangles'" },
+        "infill_sparse_density": { "maximum_value": "100" },
+        "infill_wipe_dist": { "value": "0" },
+        "inset_direction": { "value": "'inside_out'" },
+        "jerk_enabled": { "value": "True" },
+        "jerk_print": { "value": "20" },
+        "jerk_travel":
+        {
+            "maximum_value": "5",
+            "value": "5"
+        },
+        "jerk_travel_enabled": { "value": "True" },
+        "layer_height": { "value": "min(min(extruderValues('machine_nozzle_size')) / 2, 0.2)" },
+        "layer_start_x": { "value": "sum(extruderValues('machine_extruder_start_pos_x')) / len(extruderValues('machine_extruder_start_pos_x'))" },
+        "layer_start_y": { "value": "sum(extruderValues('machine_extruder_start_pos_y')) / len(extruderValues('machine_extruder_start_pos_y'))" },
+        "limit_support_retractions": { "value": "False" },
```

Comment:
this setting was removed back in version 5.3, is it needed? can we remove the setting key?
https://github.com/Ultimaker/Cura/blob/549d107ab12da6a613c96bd49b50457d9429d37a/plugins/VersionUpgrade/VersionUpgrade52to53/VersionUpgrade52to53.py#L10

Reply:
I removed all redundant settings. Will update the version now.